src/Form/Form.js

```diff
@@ -106,9 +110,9 @@ class Form extends React.Component {
 // }
 
 const mapDispatchToProps = dispatch => (
-  bindActionCreators({ setPlayerName, getQuestions }, dispatch)
+  bindActionCreators({ setPlayerName, getQuestions, isLoading }, dispatch)
```

Comment:
Still want to know whats happening under the hood with the bindActionCreators...